pkgs/by-name/ss/sss_code/package.nix

```
@@ -0,0 +1,45 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  fontconfig,
+  freetype,
+  libxcb,
+  lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sss_code";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "SergioRibera";
+    repo = "sss";
+    rev = "sss_code/v${version}";
+    hash = "sha256-rVEFdbbLghjaFaut5Xi0oK3Q1kuo3tUANfhvjnb7+3A=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "mouse_position-0.1.3" = "sha256-aYgWaozAECw+RzD2cOT7OxYzKJd2MlLsXkqW27BjSwI=";
+    };
+  };
+
+  # Removed 'sss' command due to version differences and to separate commands.
+  postInstall = ''
+    rm -f $out/bin/sss
+  '';
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ fontconfig freetype libxcb ];
+
+  meta = with lib; {
+    description = "A CLI/Lib to take amazing screenshot of code";
```

According to the [recommendations for meta attributes]( https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) description should not start with an article and avoid subjective language, hence I propose to use the following shortened description from the projects readme:
```suggestion
    description = "Libraries and tools for building screenshots in a high-performance image format";
```

---

@afh it's ready. Thank you, I changed the description and also made changes to the code because the package was updated.